swagger.yaml

```diff
@@ -10,27 +10,461 @@ basePath: "/v1"
+      tables:
+        type: array
+        items:
+          $ref: '#/definitions/FigureSection'
+      update_date:
+        type: string
+        format: 'date-time'
+        example: "2019-11-12T15:53:59.810Z"
+      links:
+        type: object
+        properties:
+          self:
+            $ref: '#/definitions/Link'
+          bulletin:
+            $ref: '#/definitions/Link'
+          versions:
+            $ref: '#/definitions/Link'
+
+  Alert:
+    type: object
+    properties:
+      date:
+        type: string
+        format: 'date-time'
+        example: "2019-09-29T23:00:00.000Z"
+      description:
+        type: string
+        description: Markdown formatted description of the alert
+        example: "The Business investment by industry and ...for the inconvenience"
+      type:
+        type: string
+        example: "alert"
+
+  Contact:
+    type: object
+    properties:
+      email:
+        type: string
+        example: "joe.bloggs@ons.gov.uk"
+      name:
+        type: string
+        example: "Joe Bloggs"
+      telephone:
+        type: string
+        example: "+44 (0)845 601 3034"
+
+  ContentSection:
     type: object
     properties:
-      message:
+      title:
         type: string
-        description: "Message returned by hello world endpoint"
-        example: "Hello, world!"
\ No newline at end of file
+        description: Title of the section
+        example: "Section Title"
+      content:
+        type: string
+        description: Markdown formatted content for the section
+        example: "Some _content_ in [Markdown](https://en.wikipedia.org/wiki/Markdown)"
+      collapse:
+        type: boolean
+        description: Signifies that the section should initially be displayed in a collapsed state
+
+  Downloads:
+    type: object
+    properties:
+      pdftables:
+        $ref: '#/definitions/Download'
+      pdf:
+        $ref: '#/definitions/Download'
+
+  Download:
+    type: object
+    properties:
+      href:
+        type: string
+        format: uri
+      size:
+        type: integer
+
+  FigureSection:
+    type: object
+    properties:
+      filename:
+        type: string
+        description: Filename of the chart / equation / image / table
+        example: "57ec16d6"
+      title:
+        type: string
+        description: Title of the chart / equation / image / table
+        example: "A Short Title"
+      uri:
+        type: string
+        description: URI for the chart / equation / image / table
+        example: "/location/of/item"
+      version:
+        type: string
+        description: Version
+        example: "2"
+
+  RelatedLink:
+    type: object
+    properties:
+      index:
+        type: integer
+      title:
+        type: string
+      uri:
+        type: string
+
+  Link:
+    type: object
+    properties:
+      href:
+        type: string
+        format: uri
+      id:
+        type: string
+    required:
+      - href
```

Comment:
I have not really considered required/optional fields, but I think we should do so. Here I know a link will always need a href so I've marked it required, but throughout the spec I'm not sure what is optional or not. Some advice on this would be helpful.

Reply:
I will add a column to the spreadsheet and comment on there as I think it will be easier.